YAXLib/YAXSerializer.cs

```diff
@@ -228,6 +228,11 @@ public YAXSerializationOptions SerializationOption
         }
 
         /// <summary>
+        /// Gets or Sets the Attribute Preprocessor callback
+        /// </summary>
+        public IAttributesPreprocessor AttributesPreprocessor { get; set; } = null;

```

Comment:
YAXLib is still in C# 5.0 era, this seems to be what's breaking the build. BTW default value for any reference type is always `null`.